python/torch_mlir/dialects/torch/importer/jit_ir/build_tools/torch_ods_gen.py

```diff
@@ -492,6 +492,9 @@ def emit_with_mutating_variants(key, **kwargs):
         emit("aten::rsub.Scalar : (Tensor, Scalar, Scalar) -> (Tensor)")
         emit("aten::gelu : (Tensor) -> (Tensor)")
         emit("aten::pow.Tensor_Scalar : (Tensor, Scalar) -> (Tensor)")
+        emit("aten::threshold : (Tensor, Scalar, Scalar) -> (Tensor)")
+        emit("aten::threshold_ : (Tensor, Scalar, Scalar) -> (Tensor)")
```

Comment:
we should use the `emit_with_mutating_variants` helper which would generate both the variant with and without `_`.